docs/data/pages.ts

```diff
@@ -6,7 +6,7 @@ import treeViewComponentApi from './tree-view-component-api-pages';
 
 const pages: MuiPage[] = [
   {
-    pathname: '/blog/mui-x-v6/',
+    pathname: '/x/whats-new',
     title: "✨ What's new in v6? ✨",
```

Comment:
```suggestion
    title: "✨ What's new in MUI X ✨",
```

Reply:
We might want to remove all the ✨. I tried it out, it feels better without:

<img width="915" alt="Screenshot 2023-09-30 at 01 31 42" src="https://github.com/mui/mui-x/assets/3165635/274460c9-d69f-4b55-a16d-9e68caf45f28">